src/Ethereum.sol

```diff
@@ -41,7 +42,9 @@ library Ethereum {
     /*** SparkDAO Addresses                                                                                         ***/
     /******************************************************************************************************************/
 
-    address internal constant SPARK_PROXY = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
+    address internal constant SPARK_PROXY         = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
+    address internal constant SPARK_REWARDS_ADMIN = 0xF649956f43825d4d7295a50EDdBe1EDC814A3a83;  // TODO: Best way to differentiate between this and MULTISIG_REWARDS?
```

Comment:
We might want a SPARK_REWARDS_MULTISIG as well to make it clear that the admin and the wallet are not necessarily the same. It is in the current setup though.

Reply:
Yes `SPARK_REWARDS_MULTISIG` is good. Also, add it to the other networks.

`MULTISIG_REWARDS` can be changed to `SPARKLEND_REWARDS_MULTISIG`